Lib/Allocator.cpp

```diff
@@ -42,7 +44,6 @@ void *operator new(size_t size, std::align_val_t align_val) {
     throw std::bad_alloc();
   ALLOCATED += size;
   {
-    Lib::TimeoutProtector tp;
```

Comment:
We don't need to protect calls to the system allocator any more as it's supposed to be thread-safe (but was *not* signal-safe).